src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -453,6 +452,7 @@ private void SetInternalTextColor(Vector4 color)
             if (color != null)
             {
                 Object.InternalSetPropertyVector4(SwigCPtr, Property.TextColor, color.SwigCPtr);
+                NotifyPropertyChanged();
```

Comment:
You need know about `NotifyPropertyChanged` 

``` c#
internal void NotifyPropertyChanged([CallerMemberName] String propertyName = "")
```
`[CallerMemberName]` is automatically filled with caller name, So if it was called in setter of TextColor, propertyName is become `TextColor`, 
but you called on `SetInternalTextColor` so propertyName is become 'SetInternalTextColor`

so it's wrong